### PR TITLE
Move us to the latest mkdocs

### PR DESCRIPTION
**Pull request by nigelbabu**
Add some changes to match the style of latest mkdocs.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -178,6 +178,5 @@ pages:
 - Google Site Verification: ./google64817fdc11b2f6b6.html
 
 theme: readthedocs
-theme_dir: glustertheme
```
Comment (prashanthpai) [inline mkdocs.yml]: Is specifying `theme_dir` no longer required or the contents of that directory also isn't required ? It contains the fix you submitted long ago to properly form canonical URL.

Comment (nigelbabu) [reply mkdocs.yml]: It appears that `theme_dir` is no longer a valid entry. Updated it with the current way of doing it.

**Comment by humblec** [comment]
@prashanthpai any further comments ?

**Comment by nigelbabu** [comment]
Right, so I just tested a merge of another patch. Without merging this patch. No other merge will build unfortunately.

**Comment by nigelbabu** [comment]
Given @prashanthpai's approval, I'm going to merge this.

**Pull request merged**